features/filiatie.feature

```diff
@@ -0,0 +1,470 @@
+            En bevat het antwoord geen property "indicatieVervallen"
+            En bevat het antwoord geen property "einddatum"
+            En bevat het antwoord geen property "indicatieSluimerend"
+            En bevat het antwoord geen property "appartementsrechtVolgnummer"
+            En bevat het antwoord geen property "bijbehorendeGrondperceelIdentificaties"
+
+        Scenario: Opvragen appartementsrecht uit splitsing
+            Als "/kadastraalonroerendezaken/A1" wordt gevraagd
+            Dan bevat het antwoord "isOntstaanUit" met waarde:    
+                """
+                [
+                    {
+                        "identificatie": "P7"
+                    }
+                ]
+                """
+            En bevat het antwoord "bijbehorendeGrondperceelIdentificaties": [ "P7" ]
+            En bevat het antwoord "appartementsrechtVolgnummer": 1
+            En bevat het antwoord geen property "indicatieVervallen"
+            En bevat het antwoord geen property "einddatum"
+            En bevat het antwoord geen property "indicatieSluimerend"
+            En bevat het antwoord geen property "isOvergegaanIn"
+            En bevat het antwoord geen property "bijbehorendeAppartementsrechtIdentificaties"
+
+        Scenario: Opvragen appartementsrecht met meerdere bijbehorende grondpercelen
+            Als "/kadastraalonroerendezaken/A3" wordt gevraagd
+            Dan bevat het antwoord "isOntstaanUit" met waarde:    
+                """
+                [
+                    {
+                        "identificatie": "P8"
+                    },
+                    {
+                        "identificatie": "P9"
+                    }
+                ]
+                """
+            En bevat het antwoord "bijbehorendeGrondperceelIdentificaties": [ "P8", "P9" ]
+            En bevat het antwoord "appartementsrechtVolgnummer": 1
+            En bevat het antwoord geen property "indicatieVervallen"
+            En bevat het antwoord geen property "einddatum"
+            En bevat het antwoord geen property "indicatieSluimerend"
+            En bevat het antwoord geen property "isOvergegaanIn"
+            En bevat het antwoord geen property "bijbehorendeAppartementsrechtIdentificaties"
+
+    Rule: Bij ondersplitsing van een appartementsrecht worden alle bijbehorende appartementsrechten getoond bij het perceel en wordt indicatieSluimerend getoond bij het sluimerende appartementsrecht
+        Scenario: Opvragen van het grondperceel na splitsing in appartementsrechten en ondersplitsing van een appartementsrecht
+            Als "/kadastraalonroerendezaken/P10" wordt gevraagd
+            Dan bevat het antwoord "isOvergegaanIn" met waarde:    
+                """
+                [
+                    {
+                        "identificatie": "A5"
+                    },
+                    {
+                        "identificatie": "A6"
+                    }
+                ]
+                """
```

Comment:
Moet hier dan ook niethet appartementsrechtVolgnummer getoond worden ?
Die geldt overal voor isOvergegaanIn en isOntstaanUit met appartementsrechten.
Of hoort dat bij de wijziging a.g.v. de structuur die nog openstaat ?

Als het appartemenstrechtVolgnummer niet getoond hoeft te worden dan staat het misschien onterecht in de OAS3 (misschien dat de discussies elkaar gekruist hebben.)